jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -1188,18 +1188,19 @@ static void checkTmpDir() {
 
 static void initSecurityManager(ClassLoader applicationClassLoader) {
 	String javaSecurityManager = internalGetProperties().getProperty("java.security.manager"); //$NON-NLS-1$
-	/*[IF JAVA_SPEC_VERSION > 11]*/
 	if ("allow".equals(javaSecurityManager)) {
```

Comment:
This method should either be annotated with `@SuppressWarnings("nls")` or `//$NON-NLS-1$` should be used consistently.